### ISSUE TITLE
allowTransparency console warning

### ISSUE THREAD

**Issue by Zashy** [interactivethings/catalog#371]
Using the responsive react specimen this warning pops up a lot. Sort of annoying.

```
Warning: React does not recognize the `allowTransparency` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `allowtransparency` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
```

**Comment by plumpNation** [comment]
It is obsolete, but would the maintainers rather a PR replacing/removing the option completely, or lowercasing it as the warning suggests. (If lowercasing the attribute in FrameComponent.js, you then get an eslint warning about the attribute not existing "react/no-unknown-property".)

Also, would you want frameBorder and scrolling options replaced with styles?

**Comment by herrstucki** [comment]
@plumpNation thank's for the nudge 😄 I went ahead and fixed it in #375 (in case you want to review my change).

**Comment by Zashy** [comment]
Awesome! Thanks for fixing this.
